leakcanary-analyzer/src/main/java/com/squareup/leakcanary/HeapAnalyzer.java

```diff
@@ -116,7 +116,8 @@ public HeapAnalyzer(ExcludedRefs excludedRefs, AnalyzerProgressListener listener
    * Searches the heap dump for a {@link KeyedWeakReference} instance with the corresponding key,
    * and then computes the shortest strong reference path from that instance to the GC roots.
    */
-  public AnalysisResult checkForLeak(File heapDumpFile, String referenceKey) {
+  public AnalysisResult checkForLeak(File heapDumpFile, String referenceKey,
+      boolean computeRetainedSize) {
```

Comment:
This is a binary incompatible change. Are you bumping the major version?

Reply:
Yes, it's a binary incompatible change, no I'm not bumping it. We've done it in the past :) 

https://github.com/square/leakcanary/blob/7988842ebd4ba54b10183d045e8b3a2af6c8f75c/leakcanary-android/src/main/java/com/squareup/leakcanary/internal/DisplayLeakActivity.java#L504

We already handle this by deleting files we can't deserialize. The idea is that people don't care about keep leaks forever and if you keep it but don't look at it straight away it'll soon be stale. So we're ok with losing local leaks on updates.

Reply:
Oh I wasn't even thinking about stored leaks. I don't care if they're lost. I'd explicitly purge them even. I'm more concerned about the fact that public API is breaking that someone could be using.

Reply:
You're right! I'll add a constructor and deprecate the old.